### ISSUE TITLE
Can't connect to API over HTTPS after upgrading to 2.8.0

### ISSUE THREAD

**Issue by xuhcc** [ipfs-shipyard/ipfs-companion#706]
OS: ArchLinux
Browser: Firefox 66.0.2 

I'm using a custom IPFS gateway (API URL `https://***:5002`). The "Redirect to gateway" switch in popup UI immediately returns back to "off" position. Browser console contains the following messages:

```
Error while ipfs.swarm.peers: ReferenceError: https is not defined backgroundPage.bundle.js:85:316595
[ipfs-companion] IPFS API is Offline: Automatic Mode: Public Gateway will be used as a fallback backgroundPage.bundle.js:181:107019
```

**Comment by lidel** [comment]
Thank you for reporting this. It seems to be a bug from https://github.com/ipfs/js-ipfs-http-client/issues/959
I will prepare bugfix release shortly.

**Comment by xuhcc** [comment]
@lidel Thank you! I just tested v2.8.0.770 and the problem is gone.

However, there is another issue that could be somehow related to this one. When I open Web UI using the "Open Web UI" menu item, page loads with error "Failed to connect to the API." It started to happen before 2.8.0, nearly a month ago (reported here https://github.com/ipfs-shipyard/ipfs-webui/issues/989).

**Comment by lidel** [comment]
@xuhcc Cool! I'll ship the fix to Stable Channel as 2.8.1

The bug with Web UI needs to be solved upstream, lets continue in https://github.com/ipfs-shipyard/ipfs-webui/issues/989
